### ISSUE TITLE
seqinfo<-() fails when @rowData is a GRangesList

### ISSUE THREAD

**Issue by lawremi** [Bioconductor/SummarizedExperiment#28]
It's not clear why `seqinfo,RangedSummarizedExperiment()` calls `update(x@rowRanges, ...)` instead of calling `seqinfo(x@rowRanges) <- value`. But the end result, when `x@rowRanges` is a GRangesList, is
```
Error in methods::slot(object, name) : 
  no slot of name "call" for this object of class "CompressedGRangesList"
```
since there is no `update()` method for a GRangesList.

**Comment by hpages** [comment]
Hi Michael,

This should be addressed in SummarizedExperiment 1.15.6 (commit 4a62d754b67a13656c1851f62fc46d0ae075395c).

H.

**Comment by lawremi** [comment]
Great, thanks.
